cms/static/js/spec/views/video_thumbnail_spec.js

```diff
@@ -149,38 +195,152 @@ define(
+                expect($thumbnailErrorEl.find('.action-text').html().trim()).toEqual(errorMessage);
+            });
+
+            it('should show error message when file is larger than maximum size', function() {
+                var errorMessage,
+                    $thumbnailErrorEl,
+                    videoView = createVideoView({}),
+                    $videoEl = videoView.render().$el,
+                    videoThumbnailView = videoView.videoThumbnailView,  // eslint-disable-line no-shadow
+                    $el = videoThumbnailView.render().$el;
+
+                videoThumbnailView.chooseFile();
+
+                // Add image to upload queue and send POST request to upload image
+                $el.find('.upload-image-input')
+                    .fileupload('add', {files: [createFakeImageFile(VIDEO_IMAGE_MAX_BYTES + 10)]});
+
+                errorMessage = 'The selected image must be smaller than ' +
+                    videoThumbnailView.getVideoImageMaxSize().humanize + '.';
+                // Verify error message
+                $thumbnailErrorEl = $videoEl.find('.thumbnail-error-wrapper .thumbnail-error');
+                expect($thumbnailErrorEl.find('.action-text').html().trim()).toEqual(errorMessage);
+            });
+
+            it('should not show error message when file is appropriate size', function() {
+                var videoView = createVideoView({}),
+                    $videoEl = videoView.render().$el,
+                    videoThumbnailView = videoView.videoThumbnailView,  // eslint-disable-line no-shadow
+                    $el = videoThumbnailView.render().$el;
+
+                videoThumbnailView.chooseFile();
+
+                // Add image to upload queue and send POST request to upload image
+                $el.find('.upload-image-input')
+                    .fileupload('add', {files: [createFakeImageFile(VIDEO_IMAGE_MIN_BYTES)]});
+
+                // Verify error not present.
+                expect($videoEl.find('.thumbnail-error-wrapper .thumbnail-error')).not.toExist();
+            });
+
+            it('should show error message when file has unsupported content type', function() {
+                var errorMessage,
+                    $thumbnailErrorEl,
+                    videoView = createVideoView({}),
+                    $videoEl = videoView.render().$el,
+                    videoThumbnailView = videoView.videoThumbnailView,  // eslint-disable-line no-shadow
+                    $el = videoThumbnailView.render().$el;
+
+                videoThumbnailView.chooseFile();
+
+                // Add image to upload queue and send POST request to upload image
+                $el.find('.upload-image-input')
+                    .fileupload('add', {files: [createFakeImageFile(VIDEO_IMAGE_MIN_BYTES, 'mov/mp4')]});
+
+
+                errorMessage = 'The selected image contains unsupported image file type. Supported file formats are ' +
```

Comment:
This image file type is not supported. Supported file types are .gif, .jpg, and .png.